lib/Figure/FigurePreview.js

```diff
@@ -39,12 +39,21 @@ const FigurePreview = ({
     );
   }
   return (
-    <img
-      title={caption}
-      src={previewSrc}
-      alt={altText}
-      className="figure__thumbnail"
-    />
+    <Fragment>
+      <img
+        title={caption}
+        src={previewSrc}
+        alt={altText}
+        className="visually-hidden"
+      />
+      <div
```

Comment:
Is the image visually hidden all the time? Is this component ever used with the image being it's native size?

Reply:
Yeah the `img` tag is always visually hidden. I am unsure if this component was ever used for the image's native size. But I am not sure it matters because it is using a div like it used to, so that would be a separate bug from before.